pixart.js

```diff
@@ -1 +1,21 @@
+var brushBox = document.querySelector(".brush");
+var inputBox = document.innerHTML.querySelector("#color-field").value;
```

Comment:
we only every call `querySelector` on `document`.

`innerHTML` is only ever used to change the contents of an element (and even then, it's very rare to use innerHTML)